app/views/stake_accounts/_explorer_stake_accounts.html.erb

```diff
@@ -0,0 +1,55 @@
+<table class='table table-block-sm mb-0'>
+  <thead>
+    <td>stake pubkey</td>
+    <td>
+      staker <br />
+      withdrawer
+    </td>
+    <td>delegated stake (SOL)</td>
+    <td>stake pool</td>
+  </thead>
+  <tbody>
+    <% @explorer_stake_accounts.each do |e_stake_acc| %>
+      <tr>
+        <td><%= e_stake_acc.stake_pubkey %></td>
+        <td>
+          <%= link_to stake_acc.staker, 
+                      url_for(account: @validator&.account, vote_account: @vote_account&.account, staker: e_stake_acc.staker) 
+          %>
+          <% if @validator %>
+            <%= link_to explorer_stake_accounts_path(staker: e_stake_acc.staker) do %>
+              <i class="fa-solid fa-up-right-from-square small ms-2"></i>
+            <% end %>
+          <% end %>
+          <br />
+          <%= link_to e_stake_acc.withdrawer,
+                      url_for(account: @validator&.account, vote_account: @vote_account&.account, withdrawer: e_stake_acc.withdrawer)
+          %>
+          <% if @validator %>
+            <%= link_to explorer_stake_accounts_path(withdrawer: e_stake_acc.withdrawer) do %>
+              <i class="fa-solid fa-up-right-from-square small ms-2"></i>
+            <% end %>
+          <% end %>
+        </td>
+        <td class="text-center">
+          <%= (stake = lamports_to_sol(e_stake_acc.delegated_stake).round(2)) == 0 ? " < 0.01" : stake %>
```

Comment:
I got:
> undefined method `/' for nil:NilClass

since `e_stake_acc.delegated_stake` is nil.

It turns out that there are many ExplorerStakeAccount records with delegated_stake value as nil.